packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts

```diff
@@ -254,8 +254,12 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
   /**
    * Whether the input field should have autocomplete enabled.
    * This tells the browser to display options based on earlier typed values.
+   *
+   * For possible values see the following links:
+   * https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values
+   * https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill
    */
-  autoComplete?: 'on' | 'off';
+  autoComplete?: 'on' | 'off' | string;
```

Comment:
Since `string` is a superset of `'on' | 'off'`, it might be better to remove the on/off part and instead just note in the description that common values are `'on'` and `'off'` but the full list of values can be found at the link.

Reply:
Sounds good, I'll change it tomorrow when I'm back in the office.

Reply:
I've updated the comment and changed the type to just be ```string```, let me know if the language needs further tweaking. (As an aside, am I supposed to resolve this conversation now,  or wait until the latest change has been OKed. Or is resolving these conversations something you would do form your side?)

Reply:
Looks good to me, thanks! I'm not sure we have a standard for who resolves conversations, so whatever seems good to you should be fine. (If you resolve a conversation and someone disagrees, they can just add another comment and un-resolve).